<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>95</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_username">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>180</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_username">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lan_public_key">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>公钥路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_public_key">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>未选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_server_addr">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_server_port">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_server_port">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>180</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_server_addr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>180</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
